RockLib.Logging.AspNetCore/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 4.0.1 - 2023-02-21
+
+#### Added
+- Updated RockLib.Logging package reference to `4.0.2`
```

Comment:
Is it `4.0.2` or `4.0.1`?

Reply:
It's `4.0.2` fixed the description above.

Reply:
@kristycurrier my bad just fixed it